README.md

```diff
@@ -1,16 +1,16 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Math Magicians
 
-> Description the project.
+> This is a single page app built with React that has 3 sections; Home, Calculator, and Quotes. The home page welcomes the user, the calculator calculates, and the quotes page displays a math quote fetched from an API.
 
 ## Built With
 
 - HTML, CSS, JavaScript, React, JSX
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://dodoburner.github.io/math-magicians/)
 
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_ on line 33 of your ReadMe, as this project is done solely. This will make your ReadMe appear more professional.